Source/WebCore/css/SelectorCheckerTestFunctions.h

```diff
@@ -432,6 +432,17 @@ ALWAYS_INLINE bool matchesFullscreenDocumentPseudoClass(const Element& element)
     return fullscreenManager && fullscreenManager->fullscreenElement();
 }
 
+ALWAYS_INLINE bool matchesInWindowFullScreenPseudoClass(const Element& element)
+{
+    if (&element != element.document().fullscreenManager().currentFullscreenElement())
+        return false;
+
+    if (auto* mediaElement = dynamicDowncast<HTMLMediaElement>(element); mediaElement && mediaElement->fullscreenMode() == HTMLMediaElementEnums::VideoFullscreenModeInWindow)
+        return true;
+
+    return false;
```

Comment:
You want to use RefPtr and there's a more concise way to write this:
```suggestion
    RefPtr mediaElement = dynamicDowncast<HTMLMediaElement>(element);
    return mediaElement && mediaElement->fullscreenMode() == HTMLMediaElementEnums::VideoFullscreenModeInWindow;
```

Reply:
Much more concise! Thank you

Reply:
You don't need a RefPtr there though.